### PR TITLE
multiple word support

### PR DESCRIPTION
**Pull request by abhishekkrthakur**
now supports multiple words and has a transform function

### CODE REVIEW

File: pycovfefy/pycovfefy.py [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
-            result += c
-            result += i
-            return result
-        if not first_consonant_after_first_vowel_found:
-            result += i
-    return result
+import string
+
+
+class Covfefy(object):
```
Comment (eyadsibai) [inline pycovfefy/pycovfefy.py]: there is no need to have a class here ...

Comment (abhishekkrthakur) [inline pycovfefy/pycovfefy.py]: done.

File: pycovfefy/pycovfefy.py [hunk 2]
```diff
@@ -31,3 +34,9 @@ def get_covfefe(word):
         if not first_consonant_after_first_vowel_found:
             result += i
     return result
+
+
+def covfefy(s):
```
Comment (eyadsibai) [inline pycovfefy/pycovfefy.py]: change s to words

Comment (eyadsibai) [inline pycovfefy/pycovfefy.py]: nice! but write a test case for this function

Comment (abhishekkrthakur) [inline pycovfefy/pycovfefy.py]: added tests and replaced s to words

**Pull request merged**